src/lib/utils/platform.js

```diff
@@ -43,6 +45,8 @@ export const isAndroid = isAndroidWeb || isAndroidNative
 // otherwise (e.g. running on native) will return false (because library wrongly returns true in that case)
 export const isBrowser = isWeb ? isBrowserWeb : false
 
+export const isEmulator = isMobileNative && DeviceInfo.isEmulator()
```

Comment:
```suggestion
export const isEmulator = once(async () => isMobileNative ? DeviceInfo.isEmulator() : false)
```

Reply:
this is already evaluated just once

Reply:
@sirpy 
1. as it's async thing, let it be a function (like in the device info library). Promise-flag isn't a good patter for the helper utility. It have to be a pre-evaluated constant or (because it's impossible in this particular case) a function
2. I wrapped it to once to be executed actually once like in your implementation.
3. anyway you're wrong here. if isMobileNative is false, the result of expression will be false. And you would try to await an boolean value instead of the promise. So, even if you want to keep your implementation, please change it to:

```javascript
export const isEmulator = isMobileNative ? DeviceInfo.isEmulator() : Promise.resolve(false)
```

Reply:
fixed pt. 3 in the commit below

Reply:
10x